src/test/java/com/wikia/webdriver/common/core/drivers/browsers/AndroidBrowser.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import io.appium.java_client.android.AndroidDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class AndroidBrowser extends BrowserAbstract {
+
+  private static AndroidDriver mobileDriver;
+
+  @Override
+  public void setOptions(){
+    DesiredCapabilities destCaps = new DesiredCapabilities();
+    destCaps.setCapability("deviceName", Configuration.getDeviceName());
+    URL url = null;
+    try {
+      url = new URL("http://" + Configuration.getAppiumIp() + "/wd/hub");
+    } catch (MalformedURLException e) {
+      PageObjectLogging.log("getAndroindInstance", e, false);
+    }
+    mobileDriver = new AndroidDriver(url, destCaps);
+
+  }
+
+  @Override
+  public WikiaWebDriver create() {
+    return new WikiaWebDriver(mobileDriver, true);
+  }
+
+  @Override
+  public void addExtension(String extensionName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1186)